documentation/docs/invoices.mdx

```diff
@@ -98,26 +98,27 @@ Content-Type: application/json
 
 ### Payment Details
 
-The Client gets the payment details of an Invoice by doing a HTTP OPTIONS of the corresponding Invoice name as URI. The Open 
-Payments Server MUST respond with either an Payment Details Response or an Error (TBD).
+The Client gets the payment details of an Invoice by doing an HTTP GET of the corresponding Invoice name as a URI, with an
```

Comment:
Instead of the additional roundtrip after the creating the invoice, can't unique payment details be returned after the `POST` creating the invoice and/or within each `GET` to the invoice? Does this warrant distinct resources?